### ISSUE TITLE
OptionsTest.java:24-28: Options are not implemented. We...

### ISSUE THREAD

**Issue by 0pdd** [zerocracy/farm#703]
The puzzle `258-bca58f1a` from #258 has to be resolved:

https://github.com/zerocracy/farm/blob/840e714a5c7dc060efee69bb1f6357956dd6da7e/src/test/java/com/zerocracy/pmo/OptionsTest.java#L24-L28

The puzzle was created by g4s8 on 02-Mar-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#703](https://github.com/zerocracy/farm/issues/703) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#703](https://github.com/zerocracy/farm/issues/703) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/951117121); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 7days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@0crat waiting #905 PR to merge

**Comment by 0crat** [comment]
> @0crat waiting #905 PR to merge ([here](https://github.com/zerocracy/farm/issues/703#issuecomment-381348723))

@g4s8 The impediment for [#703](https://github.com/zerocracy/farm/issues/703) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@da044789-3a52-46a3-9276-7cb0d37b26bd, thread: BkParallel-0
com.zerocracy.radars.github.ReSafe[76] java.io.IOException: java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML:
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1041817213">
   <created>2018-04-14T18:13:11.214Z</created>
   <type>Register impediment</type>
   <token>github;zerocracy/farm;715;381348857</token>
   <author>g4s8</author>
   <params>
      <param name="job">gh:zerocracy/farm#715</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">715</param>
      <param name="comment">381348857</param>
   </params>
</claim>

com.zerocracy.pm.Claims[111] java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML:
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1041817213">
   <created>2018-04-14T18:13:11.214Z</created>
   <type>Register impediment</type>
   <token>github;zerocracy/farm;715;381348857</token>
   <author>g4s8</author>
   <params>
      <param name="job">gh:zerocracy/farm#715</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">715</param>
      <param name="comment">381348857</param>
   </params>
</claim>

```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by g4s8** [comment]
@yegor256 looks like `0pdd` error, this puzzle was removed in #905 PR, can you close this manually?

**Comment by yegor256** [comment]
@g4s8 if the ticket is not closed, the puzzle is still in the code base. Are you sure it's gone?

**Comment by g4s8** [comment]
@yegor256 yes, I can't find it with text-search in fresh master branch:
```sh
$ ag -i "options are" ./src
src/main/java/com/zerocracy/pmo/Options.java
33: * @todo #703:30min Options are not used. Election should check
```
also I can't find it in `0pdd.xml` file generated with command `pdd --source=$(pwd) --verbose --file=/tmp/0pdd.xml`

**Comment by yegor256** [comment]
@g4s8 but it's still there: http://www.0pdd.com/p?name=zerocracy/farm

**Comment by g4s8** [comment]
@yegor256 looks like 0pdd bug, I submitted a ticket: yegor256/0pdd#205

**Comment by 0pdd** [comment]
The puzzle `258-bca58f1a` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@789b861f-9fb0-4ee3-acf5-4b943019aa6f, thread: BkParallel-9
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): Issue: zerocracy/farm#703, Action: `closed`

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1035](https://github.com/zerocracy/farm/issues/1035) is still not solved.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @g4s8[/z](https://www.0crat.com/u/g4s8), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#703](https://github.com/zerocracy/farm/issues/703) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#1185](https://github.com/zerocracy/farm/issues/1185), [#1186](https://github.com/zerocracy/farm/issues/1186), [#1187](https://github.com/zerocracy/farm/issues/1187) are still not solved; solved: [#1035](https://github.com/zerocracy/farm/issues/1035).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1185](https://github.com/zerocracy/farm/issues/1185), [#1186](https://github.com/zerocracy/farm/issues/1186) are still not solved; solved: [#1035](https://github.com/zerocracy/farm/issues/1035), [#1187](https://github.com/zerocracy/farm/issues/1187).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#1185](https://github.com/zerocracy/farm/issues/1185), [#1186](https://github.com/zerocracy/farm/issues/1186), [#1472](https://github.com/zerocracy/farm/issues/1472) are still not solved; solved: [#1035](https://github.com/zerocracy/farm/issues/1035), [#1187](https://github.com/zerocracy/farm/issues/1187).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1185](https://github.com/zerocracy/farm/issues/1185), [#1472](https://github.com/zerocracy/farm/issues/1472) are still not solved; solved: [#1035](https://github.com/zerocracy/farm/issues/1035), [#1186](https://github.com/zerocracy/farm/issues/1186), [#1187](https://github.com/zerocracy/farm/issues/1187).
